src/hotspot/share/prims/jvmtiThreadState.hpp

```diff
@@ -357,6 +358,11 @@ class JvmtiThreadState : public CHeapObj<mtInternal> {
   bool is_pending_step_for_earlyret()  { return _pending_step_for_earlyret;  }
   void process_pending_step_for_earlyret();
 
+  // For synchronization between NotifyFramePop and FramePop posting code.
+  void set_top_frame_is_exiting() { _top_frame_is_exiting = true;  }
+  void clr_top_frame_is_exiting() { _top_frame_is_exiting = false; }
+  bool is_top_frame_is_exiting()  { return _top_frame_is_exiting;  }
```

Comment:
Just noticed you are using "is" twice in this API. I'm not sure of the hotspot convention here. Seems probably the first "is" should be dropped.

Reply:
Good catch, thanks. Fixed now.